python/lsst/meas/mosaic/updateExposure.py

```diff
@@ -235,15 +230,9 @@ def applyCalib(catalog, calib, hscRun=None):
 
         if name + sigmaName in errKeys:
             errField = catalog.schema.find(name + sigmaName).field
-            if errField.getElementCount() == 1:
-                newErrField = errField.__class__(newName + sigmaName,
-                                                 "Calibrated magnitude error from %s (%s)" %
-                                                 (errField.getName(), errField.getDoc()), "mag")
-            else:
-                newErrField = errField.__class__(newName + sigmaName,
-                                                 "Calibrated magnitude error from %s (%s)" %
-                                                 (errField.getName(), errField.getDoc()), "mag",
-                                                 errField.getElementCount())
+            newErrField = errField.__class__(newName + sigmaName,
+                                             "Calibrated magnitude error from %s (%s)" %
+                                             (errField.getName(), errField.getDoc()), "mag")
             newErrKeys[newName] = mapper.addMapping(errKeys[name + sigmaName], newErrField)
```

Comment:
This code was already broken by the time I got to it: `getElementCount` was never intended to be a public method, and it isn't anymore (as of the pybind11 port, I believe).  So fixing this code would require rewriting it to avoid that check.  Given that we shouldn't need to run the new version of meas_mosaic against the outputs of older versions of the rest of the pipeline anymore, I don't think that's worthwhile.